packages/shared/src/hooks/useReferralCampaign.ts

```diff
@@ -37,14 +36,12 @@ export const campaignToReferralTargetCountMap: Record<
   ReferralCampaignKey,
   number
 > = {
-  [ReferralCampaignKey.LegoMay2023]: 5,
+  [ReferralCampaignKey.null]: 0,
```

Comment:
Can't you just leave it empty?

Reply:
It's a typescript complaints because of the Values function in the file 😅
I can try and redo it again, but was a pain so quickly did this hotfix.